byzcoin/contracts.go

```diff
@@ -29,9 +29,23 @@ var ConfigInstanceID = InstanceID{}
 // CmdDarcEvolve is needed to evolve a darc.
 var CmdDarcEvolve = "evolve"
 
-// ContractFn is the type signature of the class functions which can be
+// ContractFn is the type signature of the instance factory functions which can be
```

Comment:
Cool! Looks much better, I like it. For the file `contracts.go`, can we either:
1. split it in 
  - `contracts.go` with the interface and the `BasicContract`
  - `contractDarc.go` and `contractConfig.go` (or `contractDarcConfig.go` or better name)
2. change to order in the file (I'm very much used to navigate the file without searching) to have:
  - the interface on top
  - followed by the `BasicContract`
  - followed by `contractConfig` and `contractDarc`